src/libpgmoneta/prometheus.c

```diff
@@ -326,6 +328,51 @@ home_page(int client_fd)
    data = pgmoneta_append(data, "  <h2>pgmoneta_wal_shipping_total_space</h2>\n");
    data = pgmoneta_append(data, "  The total disk space for the WAL shipping directory of a server\n");
    data = pgmoneta_append(data, "  <p>\n");
+   data = pgmoneta_append(data, "  <h2>pgmoneta_server_timeline</h2>\n");
+   data = pgmoneta_append(data, "  The current timeline a server is on\n");
+   data = pgmoneta_append(data, "  <table border=\"1\">\n");
+   data = pgmoneta_append(data, "    <tbody>\n");
+   data = pgmoneta_append(data, "      <tr>\n");
+   data = pgmoneta_append(data, "        <td>name</td>\n");
+   data = pgmoneta_append(data, "        <td>The identifier for the server</td>\n");
+   data = pgmoneta_append(data, "      </tr>\n");
+   data = pgmoneta_append(data, "    </tbody>\n");
+   data = pgmoneta_append(data, "  </table>\n");
+   data = pgmoneta_append(data, "  <p>\n");
+   data = pgmoneta_append(data, "  <h2>pgmoneta_server_parent_tli</h2>\n");
+   data = pgmoneta_append(data, "  The parent timeline of a timeline on a server\n");
+   data = pgmoneta_append(data, "  <table border=\"1\">\n");
+   data = pgmoneta_append(data, "    <tbody>\n");
+   data = pgmoneta_append(data, "      <tr>\n");
+   data = pgmoneta_append(data, "        <td>name</td>\n");
+   data = pgmoneta_append(data, "        <td>The identifier for the server</td>\n");
+   data = pgmoneta_append(data, "      </tr>\n");
+   data = pgmoneta_append(data, "      <tr>\n");
+   data = pgmoneta_append(data, "        <td>tli</td>\n");
+   data = pgmoneta_append(data, "        <td>The current/previous timeline ID for the server</td>\n");
```

Comment:
Previous ?

Reply:
Yeah, it not only shows the parent timeline of current timeline, but also the parent timeline of previous timeline, tli is the id of the timeline whose parent we want to know about.

Reply:
Something like:
![image](https://github.com/pgmoneta/pgmoneta/assets/89931362/32e47021-42d1-477a-aee7-7d092abab255)